### DISCUSSION TITLE
How to spec FASTER server cfg

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [Foxlider/FASTER#145] [Q&A]
Is it possible? Not sure that I see that input anywhere, maybe I missed it. Thanks...

**Comment by jupster** [comment]
![image](https://github.com/Foxlider/FASTER/assets/78136088/7b10c276-b125-437f-b28d-c16d9911e16a)

If this isn't what you mean maybe word your question differently.

**Comment by mwpowellhtx** [comment]
am I able to spec the config file path itself? or at least to import it?

**Reply by jupster** [reply]
At the bottom of the image i sent their is a drop down that displays it iirc

**Reply by mwpowellhtx** [reply]
Not sure I follow, are you talking about _Load performance Preset_? That's not what I mean. i.e. for a `server_dev.cfg` file itself. As a _loaded config_, and/or at least a starting point for _FASTER_.

**Reply by jupster** [reply]
Sorry i misunderstood. No i do not belive you are able to do that with FASTER at this time.

**Reply by mwpowellhtx** [reply]
No problem, I was not clear. However, I am realizing what _FASTER_ is _"all about"_ so to speak, which composes the config, with a little assistance.
